scripts/run-dependencies.sh

```diff
@@ -41,7 +41,8 @@ docker run --rm -d \
   -e REDIS_URL=redis://redis:6379 \
   -e USE_MOCK_MODEL=true \
   $HINTR_IMAGE \
-  --results-dir=/results
+  --results-dir=/results \
+  --inputs-dir=/uploads
```

Comment:
This was the main change I wanted to get in, will need this to rehydrate locally. Working on a script to be able to run a fit and put it into a local result